lib/matplotlib/widgets.py

```diff
@@ -3051,48 +3052,53 @@ def _release(self, event):
         return False
 
     def _onmove(self, event):
-        """Motion notify event handler."""
+        """
+        Motion notify event handler.
+
+        This can do one of four things:
+        - Translate
+        - Rotate
+        - Re-size
+        - Contine the creation of a new shape
```

Comment:
```suggestion
        - Continue the creation of a new shape (e.g. next segment of a PolygonSelector.)
```

Reply:
This is a `RectangleSelector` specific docstring, so I don't think the second part is relevant